### ISSUE TITLE
read_json() with ignore_errors=true does not cope with duplicate keys

### ISSUE THREAD

**Issue by dforsber** [duckdb/duckdb#11152]
### What happens?

The JSON extension throws exception and the query fails even with `ignore_errors=true` parameter.

### To Reproduce

```shell
% cat crappy.json
{"fileSize": 9798,"totalStreamRespTimeMs": 27,"lastSyncDiffMs": 1040, "fileSize": 3266,"totalStreamRespTimeMs": 0,"lastSyncDiffMs": 307, "fileSize": 3266,"totalStreamRespTimeMs": 0,"lastSyncDiffMs": 307, "fileSize": 3266,"totalStreamRespTimeMs": 0,"lastSyncDiffMs": 307}
{"fileSize": 13064,"totalStreamRespTimeMs": 1,"lastSyncDiffMs": 1409}
{"crap": 13064,"tmeMs": 1,"lastSy": 1409, "blaa": "bar"}
{"errorsToo","notparsing","badJSON"}
% duckdb -s "FROM read_json_auto('crappy.json', ignore_errors=true);"
Error: Invalid Input Error: Duplicate key "fileSize" in object {"fileSize":9798,"totalStreamRespTimeMs":27,"lastSyncDiffMs":1040,"fileSize":3266,"totalStreamRespTimeMs":0,"lastSyncDiffMs":307,"fileSize":3266,"totalStreamRespTimeMs":0,"lastSyncDiffMs":307,"fileSize":3266,"totalStreamRespTimeMs":0,"lastSyncDiffMs":307}
% echo $? 
1
```

### OS:

OSX

### DuckDB Version:

v0.10.1-dev1244 0f4a300465

### DuckDB Client:

cli

### Full Name:

Dan Forsberg

### Affiliation:

BoilingData

### Have you tried this on the latest [nightly build](https://duckdb.org/docs/installation/?version=main)?

I have tested with a nightly build

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] Yes, I have

**Comment by hannes** [comment]
The ignore errors refers to parsing errors, not integrity constraint violations.

**Comment by dforsber** [comment]
> The ignore errors refers to parsing errors, not integrity constraint violations.

Yes, I guessed something like that :). 

If NDJSON can have crap inside it so that DuckDB can ignore the lines, why make it throw when there is duplicate key?

Is the JSON parser actually passing duplicate keys "upwards" and if so, it is not producing correct JSON.

E.g. node JSON.parse() copes fine with duplicate keys, and provides back the last value.

**Comment by codekoala** [comment]
I encountered this problem a few minutes ago, literally the first time I ran duckdb. I can confirm that the newly released version 0.10.2 resolves the problem in my case. 🎉

**Comment by dforsber** [comment]
Closing this, resolved, thank you.
